Resculpt/Resculptable.cs

```diff
@@ -104,6 +120,14 @@ public sealed class Resculptable : KMonoBehaviour {
 				var button = new KIconButtonMenu.ButtonInfo(icon, text, OnResculpt,
 					PAction.MaxAction, null, null, null, ResculptStrings.RESCULPT_TOOLTIP);
 				Game.Instance?.userMenu?.AddButton(gameObject, button);
+
+				var rotationButton = new KIconButtonMenu.ButtonInfo("action_direction_both",
```

Comment:
Nit: Button is created and immediately destroyed if the object cannot be rotated, consider allocating only inside the `if (Rotatable != null)`

Reply:
moved inside if()